NetSSL_OpenSSL/include/Poco/Net/NetSSL.h

```diff
@@ -73,7 +73,7 @@
 	#endif
 
 	#if !defined(POCO_NO_AUTOMATIC_LIBS)
-		#if !defined(POCO_EXTERNAL_OPENSSL)
+		#if defined(POCO_EXTERNAL_OPENSSL)
```

Comment:
proper fix is [in develop](https://github.com/pocoproject/poco/commit/38c4e35c43a5b377c4e1db1fc1da1dca57c882f8#diff-424dc05c08fae83b2e68334e4a423b69), merge develop into your branch and then we'll merge it to develop